controller/bookmark.js

```diff
@@ -41,6 +42,24 @@ module.exports = {
       .finally(() => next())
   },
 
+  getBookmarkByTag: (req, res, next) => {
+    
+    const id = req.params.id
```

Comment:
we would also need to refactor this line to get the ```tag``` if we change the route param